samples/wifi/thread_coex/prj.conf

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+# WPA supplicant
+CONFIG_WPA_SUPP=y
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_LOG=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_DHCPV4=y
+
+CONFIG_NET_IF_UNICAST_IPV4_ADDR_COUNT=2
+CONFIG_NET_MAX_CONTEXTS=8
+CONFIG_NET_CONTEXT_SYNC_RECV=y
+
+CONFIG_INIT_STACKS=y
+
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_INIT_TIMEOUT=0
+
+# Memories
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_NET_TX_STACK_SIZE=4096
+CONFIG_NET_RX_STACK_SIZE=4096
+
+# Debugging
+CONFIG_STACK_SENTINEL=y
+CONFIG_DEBUG_COREDUMP=y
+CONFIG_DEBUG_COREDUMP_BACKEND_LOGGING=y
+CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
+
+# Kernel options
+CONFIG_ENTROPY_GENERATOR=y
+
+# Logging
+CONFIG_LOG=y
+CONFIG_PRINTK=y
+CONFIG_POSIX_CLOCK=y
+
+# Benchmarking
+CONFIG_NET_ZPERF=y
+CONFIG_POSIX_MAX_FDS=16
+
+# Optimized networking settings for performance
+CONFIG_NET_TC_TX_COUNT=1
+CONFIG_HEAP_MEM_POOL_SIZE=200000
+CONFIG_NET_BUF_DATA_SIZE=1100
+
+# Below Wi-Fi configs need to be modified based on security
+CONFIG_WIFI_MGMT_EXT=y
+CONFIG_WIFI_CREDENTIALS=y
+CONFIG_WIFI_CREDENTIALS_STATIC=y
+CONFIG_WIFI_CREDENTIALS_STATIC_SSID="Myssid"
+CONFIG_WIFI_CREDENTIALS_STATIC_PASSWORD="Mypassword"
+CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_OPEN=y
```

Comment:
This isn't needed anymore, Wi-Fi Connection + OT fix is merged to NCS. If you see crypto issues in Thread join please enable `CONFIG_MBEDTLS_MD_C=y` explicitly.

Reply:
(1) If I remove the SSID, PASSWORD etc config parameters in prj.conf then where should I update the SSID and password details to run the tests?
(2) Following are the results when the thread coexistence tests are run with security set to PSK.
        WLAN connection failed even after adding CONFIG_MBEDTLS_MD_C=y in prj.conf and 
        CONFIG_MBEDTLS_SHA1_C=y in overlay-openthread.conf.
     
        Thread connection and throughput are successful with or without the above mentioned updates.

Reply:
1. The SSID and password are needed, I was only referring to line #66.
2. Please share the WPA supplicant debug logs, `shell` sample with `overlay-openthread.conf` is working fine with security and also the `.config`

Reply:
Oh sorry. I forgot to enable the security mode in the AP configuration while testing with PSK.
The test results are as below for the test case WLAN and thread in client roles and WLAN is operating in 2.4GHz. All the tests are successful, and results are as expected.

WLAN only: Both connection and throughput are successful.
Thread only: Both connection and throughput are successful.
WLAN and Thread, coex disabled: Both WLAN and Thread throughputs are successful.
WLAN and Thread, coex enabled: Both WLAN and Thread throughputs are successful.